assets/js/components/Carousel.js

```diff
@@ -41,7 +41,6 @@ module.exports = class Carousel {
     this.updateWidth();
     this.updateControlPanel(this.currentSlide);
     this.togglePlay(this.$toggler);
-    this.$toggler.focus();
```

Comment:
Doesn't seem to make a difference, but this stops `autofocus` working otherwise.